tests/main/completion/task.yaml

```diff
@@ -1,7 +1,9 @@
 summary: Check different completions
 
-# ppc64el disabled because of https://github.com/snapcore/snapd/issues/2502
-systems: [-ubuntu-core-16-*, -ubuntu-*-ppc64el]
+systems:
```

Comment:
(nitpick) this change looks unrelated. Fine to have it, just point this out. I guess initially debian was disabled here?